src/protocol/reveal_additive_binary.rs

```diff
@@ -63,7 +64,7 @@ mod tests {
         let mut rng = rand::thread_rng();
 
         let world: TestWorld = make_world(QueryId);
-        let ctx = make_contexts::<Fp31>(&world);
+        let ctx = make_contexts::<Fp2>(&world);
```

Comment:
It's weird that this doesn't matter =). Honestly, let's keep it at Fp31 for the reasons in my comment above.